### PR TITLE
Removing rpc feature from composer

### PR DESCRIPTION
**Pull request by dsavitskiy**

### CODE REVIEW

**Review by tiagolobocastro — APPROVED** [review]

**Review by Abhinandan-Purkait — APPROVED** [review]

**Comment by dsavitskiy** [comment]
here's PR for mayastor-api repo: https://github.com/openebs/mayastor-api/pull/20

**Comment by dsavitskiy** [comment]
In lastest push:
* RPC support removed from composer
* mayastor-api submodule removed

File: composer/src/composer.rs [hunk 1]
```diff
@@ -648,6 +649,14 @@ impl Builder {
         ))
     }
 
+    /// add a debug io-engine container with a name
+    pub fn add_container_dbg(self, name: &str) -> Builder {
```
Comment (mtzaurus) [inline composer/src/composer.rs:653]: Who uses this new function ?

Comment (dsavitskiy) [reply composer/src/composer.rs:653]: Data plane.
Data plane's version Composer added a debug binary with add_container(). In this version of Composer, add_container() adds a release binary. Besides that, Build has from_dbg() method.

**Review by tiagolobocastro — APPROVED** [review]

**Pull request merged**